docker-prune-cron/README.md

```diff
@@ -0,0 +1,64 @@
+# Docker Image Pruning Cron Job for Dev server
+
+This README provides instructions for setting up a cron job to regularly prune old Docker images on a Dev server. Pruning Docker images helps to manage disk space effectively by removing unused images.
+
+## Prerequisites
+
+- SSH access to the Dev server.
+
+## Steps to Set Up the Cron Job
+
+### 1. Access the Dev Server
+
+SSH into your Dev server:
+
+```sh
+ssh azureuser@ui-1.dev.codeforafrica.org
+```
+
+
+### 2. Set Up the Cron Job
+
+Edit the cron table to schedule the pruning job every 4 hours.
+
+Open the crontab for editing:
+
+```sh
+crontab -e
+```
+
+Add the following line to schedule the job:
+
+```
+0 */4 * * * /home/azureuser/prune_docker_images.sh
```

Comment:
Normally you don't want scripts laying around in a home folder. You usually put them in say ~/.local/bin, etc.

But since we're going to develop this further to be part of Dokku plugin ecosystem, I guess we can leave it here for now.